src/otx/engine/hpo/hpo_api.py

```diff
@@ -194,10 +192,10 @@ def _get_default_search_space(self) -> dict[str, Any]:
 
     @staticmethod
     def _make_lr_search_space(optimizer_callable: OptimizerCallable) -> dict[str, Any]:
```

Comment:
```suggestion
    def _make_lr_search_space(optimizer_callable: OptimizerCallableSupportHPO) -> dict[str, Any]:
```
`optimizer_callable` should be `OptimizerCallableSupportHPO` right?

Reply:
That change will invoke a `mypy` error. Type guarding is conducted by the following lines